packages/core/src/app/customer/CheckoutButtonContainer.tsx

```diff
@@ -27,22 +37,63 @@ const CheckoutButtonContainer: FunctionComponent<CheckoutButtonsListOnTopProps &
         },
     } = checkoutState;
     const config = getConfig();
+    const methodIds = config?.checkoutSettings.remoteCheckoutProviders ?? [];
+
+    useEffect(() => {
+        const changeState = setTimeout(()=>{
+            if(!isInitializingCustomer()) {
+                setIsLoading(false);
+            }
+        },100);
+
+        return () => {
+            clearTimeout(changeState);
+            setIsLoading(true);
```

Comment:
Why do we need to set the `isLoading` state to true when the component in unmounted?

Reply:
Restore the state to `loading` whenever `isInitializingCustomer` is changing.
But since I put a delay above, this seems unnecessary.